### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by MavOmidK**
Finished the design for the basic home page.
Made home page the root.
Removed Page1/Page2 from the menu.

Added cordova dependencies to project (to run on emulator)

### CODE REVIEW

File: Keanu/config.xml [hunk 1]
```diff
@@ -1,39 +1,39 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<?xml version='1.0' encoding='utf-8'?>
```
Comment (aagavin) [inline Keanu/config.xml:1]: Why did you change the quotes in this file?

File: Keanu/src/app/app.component.ts [hunk 2]
```diff
@@ -2,9 +2,13 @@ import { Component, ViewChild } from '@angular/core';
 import { Nav, Platform } from 'ionic-angular';
 import { StatusBar, Splashscreen } from 'ionic-native';
 
+// Default Pages (Leave alone for now)
 import { Page1 } from '../pages/page1/page1';
```
Comment (aagavin) [inline Keanu/src/app/app.component.ts]: Go ahead and remove these pages

**Review by aagavin — CHANGES_REQUESTED** [review]

File: Keanu/config.xml (hunk 1 above)
Comment (MavOmidK) [reply Keanu/config.xml:1]: I didn't do this, which is weird. Could've been something that my ionic did?

File: Keanu/src/app/app.component.ts (hunk 2 above)
Comment (MavOmidK) [reply Keanu/src/app/app.component.ts]: Weird error when removing Page 1 and Page 2. Main.js would complain and break the program. Did this temp fix for now.

**Review by aagavin — APPROVED** [review]
looks good
